src/include/add-on/add-on-workflow.rb

```diff
@@ -224,37 +217,31 @@ def MediaSelect
         sources_after = Pkg.SourceGetCurrent(false)
         Builtins.y2milestone("Sources with new one added: %1", sources_after)
 
-        # bnc #393011
-        # AddOnProduct::src_id must be set to the latest source ID
-        src_id_found = false
+        @added_repos = []
 
-        Builtins.foreach(sources_after) do |one_source|
-          if !Builtins.contains(sources_before, one_source)
-            AddOnProduct.src_id = one_source
-            Builtins.y2milestone("Added source ID is: %1", AddOnProduct.src_id)
-            src_id_found = true
-            raise Break
-          end
+        sources_after.each do |source|
+          next if sources_before.include?(source)
```

Comment:
I think this deserve a comment as it is not so clear. Is it mean that sources that are already in system is skipped? So it is not mentioned in newly added sources?